README.md

```diff
@@ -346,18 +347,26 @@ ignored.
 | `alerting.twilio`      | Settings for alerts of type `twilio`. <br />See [Configuring Twilio alerts](#configuring-twilio-alerts).                     | `{}`    |
 | `alerting.custom`      | Configuration for custom actions on failure or alerts. <br />See [Configuring Custom alerts](#configuring-custom-alerts).    | `{}`    |
 
-
 #### Configuring Discord alerts
-| Parameter                        | Description                                                                                | Default       |
-|:---------------------------------|:-------------------------------------------------------------------------------------------|:--------------|
-| `alerting.discord`               | Configuration for alerts of type `discord`                                                 | `{}`          |
-| `alerting.discord.webhook-url`   | Discord Webhook URL                                                                        | Required `""` |
-| `alerting.discord.default-alert` | Default alert configuration. <br />See [Setting a default alert](#setting-a-default-alert) | N/A           |
+
+| Parameter                                  | Description                                                                                | Default       |
+|:------------------------------------------ |:------------------------------------------------------------------------------------------ |:------------- |
+| `alerting.discord`                         | Configuration for alerts of type `discord`                                                 | `{}`          |
+| `alerting.discord.webhook-url`             | Discord Webhook URL                                                                        | Required `""` |
+| `alerting.discord.default-alert`           | Default alert configuration. <br />See [Setting a default alert](#setting-a-default-alert) | N/A           |
+| `alerting.discord.overrides`               | List of overrides that may be prioritized over the default configuration                   | `[]`          |
+| `alerting.discord.overrides[].group`       | Endpoint group for which the configuration will be overridden by this configuration        | `""`          |
+| `alerting.discord.overrides[].webhook-url` | Teams Webhook URL                                                                          | `""`          |
 
 ```yaml
 alerting:
   discord: 
     webhook-url: "https://discord.com/api/webhooks/**********/**********"
+    # You can also add group-specific to keys, which will 
+    # override the to key above for the specified groups
+    overrides:
+      - group: "core"
+        webhook-url: "https://discord.com/api/webhooks/************"
```

Comment:
I don't think there's a need to explicitly show the `alerting.*.overrides` configuration, as it will create unnecessary noise in the documentation (as you can see, I did the same for `alerting.discord.default-alert`

Reply:
https://github.com/TwiN/gatus/pull/271/commits/75c34d152d3adfa5cdb5e393ee45e8bdb3693b78